src/app/app.tsx

```diff
@@ -14,25 +18,48 @@ import { ToastsProvider } from '@app/features/toasts/toasts-provider';
 import { AppRoutes } from '@app/routes/app-routes';
 import { persistor, store } from '@app/store';
 
+import localConfig from '../../config/wallet-config.json';
 import './index.css';
+import { useCurrentNetwork } from './store/networks/networks.selectors';
 
 const reactQueryDevToolsEnabled = process.env.REACT_QUERY_DEVTOOLS_ENABLED === 'true';
 
+function ConnectedApp() {
+  const network = useCurrentNetwork();
+  return (
+    <QueryClientProvider client={queryClient}>
+      <LeatherQueryProvider
+        client={queryClient}
+        network={network}
+        environment={{
+          env: WALLET_ENVIRONMENT,
+          github: {
+            org: GITHUB_ORG,
+            repo: GITHUB_REPO,
+            branchName: BRANCH_NAME,
+            localConfig: localConfig as any,
```

Comment:
We don't need to make this change now, but think the remote config should be separate